vaultutil/pkg/vault/wire.go

```diff
@@ -0,0 +1,29 @@
+//go:build wireinject
```

Comment:
This sort of similarly feels like it deserves a `vaultutil/pkg/test` all to its own, not sure. I could probably go either way here.

Reply:
I originally wanted to put more of the testing helpers in a `test` package, but the `github.com/mitchellh/go-testing-interface` dependency used in the vault test suite causes issues with that. It's not too hard to workaround it, but it's messy and not worth doing. Regardless, the new `test` package has the wire gen, which is sufficient for now.